sqlalchemy_ibmi/base.py

```diff
@@ -800,9 +801,19 @@ def create_connect_args(self, url):
 
         try:
             opts['Naming'] = str(util.strtobool(opts['use_system_naming']))
-        except (ValueError, KeyError):
+        except (KeyError):
             opts['Naming'] = '0'
+        except (ValueError):
```

Comment:
```suggestion
        except KeyError:
            opts['Naming'] = '0'
        except ValueError:
```

The parenthesis are no longer needed now that they've been separated.

Reply:
Got it.  Thanks!

Reply:
Looks like you fixed the one but not the other.